django_lti_tool_provider/tests/test_views.py

```diff
@@ -133,36 +141,20 @@ def test_no_session_given_incorrect_payload_throws_bad_request(self, _):
         self.assertIn("Invalid LTI Request", response.content)
 
     def test_no_session_correct_payload_processes_lti_request(self, patched_send_lti_received):
-        with self.assertRaises(LtiUserData.DoesNotExist):
-            LtiUserData.objects.get(user=self.user)  # precondition check
+        # Precondition check
+        self.assertFalse(LtiUserData.objects.all())
 
         response = self.send_lti_request(self.get_correct_lti_payload())
 
-        self._verify_lti_created(self.user, self._data)
-        self._verify_redirected_to(response, self._url_base + self.DEFAULT_REDIRECT)
-        self._verify_lti_updated_signal_is_sent(patched_send_lti_received, self.user)
-
-    @ddt.data('GET', 'POST')
-    def test_session_set_processes_lti_request(self, method, patched_send_lti_received):
```

Comment:
@e-kolpakov This functionality is at odds with the fix for dalite, since it means a user can exist outside of LTI and be handled by it in the LTI requests. Fixing this won't be a small task, as being able to signal to the app that we want to let these cases go through would probably mean changing the authentication manager object to make it clear this should be permitted.

I'd rather add this functionality back in once we need it rather than try to do it now. If this use case comes up in practice, we can handle it then.